contracts/stellar-interchain-token/src/tests/testdata/transfer_from_succeeds.golden

```diff
@@ -0,0 +1 @@
+Vec(Ok((Contract(CAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAMDR4), Vec(Ok(Symbol(transfer)), Ok(Address(obj#451)), Ok(Address(obj#453))), I128(400))))
```

Comment:
this is printing `obj` ids instead of the values. Format it differently, might need to convert to/from val

Reply:
you could also define the token events using `IntoEvent` but only within the tests for testing purposes. So the contract continues using the TokenUtils but the tests benefit from a well-typed event struct